### PR TITLE
Add blog post: how we measure coaching engagements

### PR DESCRIPTION
**Pull request by beechnut**
## Pull request summary

Adds the blog post on how we measure coaching engagements.

## How to review

After approving and BEFORE merging, please let me know so I can update the publish date first.

Closes https://github.com/18F/blog-drafts/issues/903

### CODE REVIEW

**Comment by alexsobledotgov** [comment]
## 👁️ 👁️ Preview link

https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/beechnut/add-coaching-blog/2024/11/22/how-we-measure-coaching/

File: content/posts/2024-11-22-how-we-measure-coaching.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: How we measure the success of coaching engagements
+authors:
+  - matt-cloyd
+date: 2024-11-22
```
Comment (alexsobledotgov) [inline content/posts/2024-11-22-how-we-measure-coaching.md:5]:
```suggestion
date: 2024-12-10
```

suggesting publish next week on Tues

**Comment by edwintorres** [comment]
> ## 👁️ 👁️ Preview link
> https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/beechnut/add-coaching-blog/2024/11/22/how-we-measure-coaching/
New preview link
https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/beechnut/add-coaching-blog/2024/12/10/how-we-measure-coaching/

**Review by edwintorres — APPROVED** [review]
LGTM!

**Pull request merged**